### PR TITLE
[FIX] Script injection is unreliable

### LINKED ISSUE #145: [bug]: script injection is unreliable
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Recently, there have been a flurry of reports about scripts not injecting reliably. I haven't done a binary search to find the breaking commit yet, but my suspicion is that this popped up in 723fd82bf353c5d66a5c6da9a029d299562c7e4e

### Expected behavior

Scripts to inject the first time GeoFS is loaded.

### Actual behavior

Scripts take multiple reloads to inject, and sometimes don't all inject.

### Operating System

Windows 10 (no version number as of yet)

### Browser

Chrome (no version number as of yet)

### Version

3.3.1

### PR DESCRIPTION
**Pull request by Guy-Adler**
Fixes #145

The problem is most probably with the caching [this line](https://github.com/GeoFS-Extensions/autoland/blob/ac38d50b997f1bd483d54c0672491216639a8f52/extension/source/background.ts#L97).
The problem, as @nicolas377 said, was caused by the rest of the code not waiting for the async code. To solve it, I moved it into the `.then()` callback.
This should be tested extensively, but my tests showed it works.

Please fill in this template.

- [x] Use a meaningful title for the pull request.
- [x] Test the change. (Compile and use the built extension.)
- [x] Run `npm run build`.
- [x] Run `npm test`.

### CODE REVIEW

**Comment by nicolas377** [comment]
it's possible that [this line](https://github.com/GeoFS-Extensions/autoland/blob/ac38d50b997f1bd483d54c0672491216639a8f52/extension/source/background.ts#L97) is causing an issue on the startup of the service worker.
As demonstrated in the image below, the options are not loaded synchronously.
image
The service worker starts up when the callback for the new tab is opened, so that might cause the options to not be loaded in time, and so the scripts get turned off by default.
I'm not sure about that though, but this code seems to be a good replacement.
image

**Comment by nicolas377** [comment]
Could you `npm run suite`? We need package and navdata updates, and I can't get to my development environment for this week.

**Comment by nicolas377** [comment]
it works on v17, and it should work on v16 LTS (16.13.1)

File: extension/source/ui/popup/popup.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
         Spoilers Arming
       </button>
     </div>
-    <div id="spoilerarmingholder">
+    <div id="keyboardmappingholder">
```
Comment (Guy-Adler) [inline extension/source/ui/popup/popup.html:22]: This had the wrong id, it might have also caused a problem we weren't aware of

Comment (nicolas377) [reply extension/source/ui/popup/popup.html:22]: Thanks for fixing that!

**Review by nicolas377 — APPROVED** [review]
LGTM! Thanks for this! I'll take the publication process from here.

**Comment by nicolas377** [comment]
@Guy-Adler could you build this and post it to the testers?

**Comment by Guy-Adler** [comment]
no problem

**Pull request merged**